### DISCUSSION TITLE
Error in restart run using Noah MP 3.6

### DISCUSSION THREAD

**Discussion by nchawang** [NASA-LIS/LISF#1353] [Q&A] [answered]
Greetings,

I had completed the cold start run for LIS from 2012 to 2014. for the restart run, i was going through the 2nd iteration for the same period, when giving the run i got the following error:

"undefined value found for forcing variable rivsto in NoahMP36
for tile 1 latitude = 5.05000019  longitude 85.3500061
[ERR] endrun is being called"

The thing is when i run the restart run in serial, there is no issues, the restart run goes as usual. Im not sure why this is happening. (the earlier cold start was ran in parallel)

Grateful for any help
Thanks!

**Comment by dmocko** [comment, marked as answer]
Hi @nchawang 

Thanks for the question.  This problem has already been discussed and answered in this thread: https://github.com/NASA-LIS/LISF/discussions/972

What LIS release are you using?  And can you use Noah-MP-4.0.1 LSM instead?

**Reply by nchawang** [reply]
Hi @dmocko 

Thankyou for the thread link, it is now working. i switched from LISF 7.3 to 7.4.

Thanks!
